place_method:
  type: ground_truth
  lc_distance_threshold: 4
  lc_recent_lockout_s: 20
keypoint_method:
  type: ground_truth
descriptor_method:
  type: ground_truth
match_method:
  type: ground_truth
pose_method:
  type: ground_truth
lc_frame_lockout_s: 20
place_match_threshold: -4
strict_keypoint_evaluation: false
display_method:
  type: ros
  display_place_matches: false
  display_keypoint_matches: false
  display_inlier_keypoint_matches: false
